Add avatar feature

kleyv opened the pull request:
I added Cloudinary and Active_storage gems and changed the seed so after merged please
```bash
git co master
git pull origin master
bundle install
rails db:migrate db:seed
```

kleyv committed:
Add cloudinary gem

Please `bundle install`

kleyv committed:
Setup active_storage and image association to user model

I installed active_storage, ran the migration and changed the
necessary files in the config.
Also attached :photo to the user model as an association.

Tiago-Palhoca committed:
Added design elements to home.scss

- Added 2 types of country cards (colapsed and expanded)
- Added additional buttons

Tiago-Palhoca committed:
Merge from master

Tiago-Palhoca committed:
Merge branch 'master' into add-design-elements

kleyv committed:
Add image upload and edit functions

Added the function to add an image when signing up and to change
it once the account is created.
I also updated the seeds to be created with seeds.
Finally I edited the navbar so that the avatar displayed is the
current_user.photo.

Tiago-Palhoca committed:
Updated the footer

- Reduced the height of the footer in the components.scss
- Decreased the padding

Tiago-Palhoca commented on app/controllers/application_controller.rb:
```diff
@@ -3,6 +3,7 @@ class ApplicationController < ActionController::Base
-    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :phone_number, :email, :country_id, :password])
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:photo, :first_name, :last_name, :phone_number, :email, :country_id, :password])
+    devise_parameter_sanitizer.permit(:account_update, keys: [:photo, :email, :password])
   end
```
Do these params sanitizers all run, or just one of them?
I don't understand it completly, but if you tested it, I trust you...

kleyv replied:
First line is for signing up (new and create) and the second line is for editing your account (patch). That's how I understand them. @raffoz @Flott2410 ??

kleyv commented:
Do not merge yet. Work in progress.

raffoz committed:
Update _home.scss

raffoz committed:
Update _home.scss

raffoz committed:
Merge pull request #14 from Flott2410/add-design-elements

Add design elements

kleyv committed:
Require users photo when signing up

I changed the user model to validate the presence of a photo and
the form on the sign up view to require it.

Flott2410 committed:
Merge branch 'add-avatar-feature' of github.com:Flott2410/falcony into add-avatar-feature

Flott2410 committed:
changed the seed file to first user.new then user.save
